tables/isolate_mutations.d/goes22-isomuts.csv

```diff
@@ -0,0 +1,6 @@
+﻿iso_name,gene,position,amino_acid,count,total
+Nsp6:del106+del107+del108,nsp6,106,del,NULL,NULL

```

Comment:
@zachandfox please convert the mutation list back to full sequence, you can use the script provided here: https://gist.github.com/philiptzou/985c89ff87987c46c2df28b74e4c4fa5, then use the analysis program to fetch the mutation list. I read the figure seems we can get 5 dates' sequences. The purpose of doing this is to record the mutation history.

We also name the isolates in selection studies using the relative date they're isolated or using the order number to order them. We don't use the combination of mutations to name them because they're full sequences not pseudotype virus.